skimage/feature/texture.py

```diff
@@ -291,3 +295,154 @@ def local_binary_pattern(image, P, R, method='default'):
+                        lbp_code=0,
+                        color_greater_block=[1, 1, 1],
+                        color_less_block=[0, 0.69, 0.96],
+                        alpha=0.5
+                        ):
+    """Multi-block local binary pattern visualization.
+
+    Blocks with higher sums are colored with transparent white rectangles,
+    whereas blocks with lower sums are colored cyan.
+    The colors can also be specified.
+    Opacity of visualization is controlled with `alpha` parameter.
+
+    Parameters
+    ----------
+    img :
+        Image on which to visualize the pattern.
+    x : int
+        X-coordinate of top left corner of a rectangle containing feature.
+    y : int
+        Y-coordinate of top left corner of a rectangle containing feature.
+    width : int
+        Width of one of 9 equal rectangles that will be used to compute
+        a feature.
+    height : int
+        Height of one of 9 equal rectangles that will be used to compute
+        a feature.
+    lbp_code : int
+        The descriptor of feature to visualize. If not provided,
+        the descriptor with 0 value will be used.
+    color_greater_block : list of 3 floats
+        Floats specifying the color for the block that
+        has greater intensity value. They should be
+        in the range [0, 1]. Corresponding values define
+        (R, G, B) values. Default value is white [1, 1, 1].
+    color_greater_block : list of 3 floats
+        Floats specifying the color for the block that
+        has greater intensity value. They should be
+        in the range [0, 1]. Corresponding values define
+        (R, G, B) values. Default value is cyan [0, 0.69, 0.96].
+    alpha : float
+        Value in the range [0, 1] that specifies opacity of
+        visualization. 1 - fully transparent, 0 - opaque.
+
+    Returns
+    -------
+    output : ndarray of float
+        Image with visualization.
+
+    References
+    ----------
+    .. [1] Face Detection Based on Multi-Block LBP
+           Representation. Lun Zhang, Rufeng Chu, Shiming Xiang, Shengcai Liao,
+           Stan Z. Li
+           http://www.cbsr.ia.ac.cn/users/scliao/papers/Zhang-ICB07-MBLBP.pdf
+    """
+
+    # Default colors for regions.
+    # White is for the blocks that are brighter.
+    # Cyan is for the blocks that has less intensity.
+    color_greater_block = np.asarray(color_greater_block, dtype=np.double)
+    color_less_block = np.asarray(color_less_block, dtype=np.double)
+
+    # Copy array to avoid the changes to the original one.
+    output = np.copy(img)
+
+    # As the visualization uses RGB color we need 3 bands.
+    if len(img.shape) < 3:
+        output = gray2rgb(img)
+
+    # Colors are specified in floats.
+    output = img_as_float(output)
+
+    # Offsets of neighbour rectangles relative to central one.
+    # It has order starting from top left and going clockwise.
+    neighbour_rect_offsets = ((-1, -1), (0, -1), (1, -1),
```

Comment:
You could `np.array()` this tuple of tuples into a 2D ndarray, then multiply x and y by height and width, respectively, prior to the `for` loop below.

The motif `for elements in ndarray_2d:` iterates along the first axis, so it should work the same as a tuple of tuples.